<Action id="60359" issue="40129" author="sykesm" type="comment" body=" When searching, I see several hits - mosts closed as &quot;invalid&quot; or &quot;won&apos;t do&quot;. Makes me think something is truly amiss." created="2019-05-24 14:14:00.0" updateauthor="sykesm" updated="2019-05-24 14:14:00.0"/>
<Action id="60364" issue="40129" author="guoger" type="comment" created="2019-05-24 16:08:17.0" updateauthor="guoger" updated="2019-05-24 16:08:17.0"> <body><! CDATA fixed as part of https://gerrit.hyperledger.org/r/c/fabric/+/31524/4  and quote commit msg here:  {quote}It also fixes a flake, which is contributed by several factors: 1) if data accumulates too fast, while previous snapshotting is still in progress, there is mechanism to prevent excessive snapshotting, then the final number of snapshot files is not deterministic 2) size of block is not big enough to trigger snapshotting for each of them  It is fixed by: 1) assert new snapshot file being created before submitting next tx 2) increase tx size  This flake was hit rarely before, because: 1) full Fabric tx flow was performed, which is slow 2) the tx size was big enough{quote}  ></body> </Action>
